cmd/dep/testdata/harness_tests/init/glide/transitive/trans-trans-conflict/initial/glide.yaml

```diff
@@ -0,0 +1,6 @@
+package: .
+import:
+- package: github.com/chriswhelix/deptestglide1
+  version: gt100
+- package: github.com/chriswhelix/deptestglide2
+  version: eq081
```

Comment:
sorry, i'm just seeing this now. unfortunately, i don't have a lot of guidance - assuming i'm reading that dump correctly, it does appear that those constraints are disjoint, which should cause a failure. (gps has scads of tests verifying that failures do occur when faced with disjoint constraints).

i might instrument something a little higher up, e.g. in `bridge.go`, to see what constraints are actually coming back from `sm.GetManifestAndLock()`